objc/README.md

```diff
@@ -427,6 +435,28 @@ if ([someObject boolValue] == NO)
 
 -----
 
+Complex if statements should be assigned to a meaningfull variable, it allows easier maintenance.
```

Comment:
One l to many in meaningful(l). Should probably rephrase the last part of the sentence to "it makes maintenance easier"